plugin/plugin_package.go

```diff
@@ -34,14 +34,23 @@ const (
 )
 
 type BuildTarget struct {
-	OS   string `json:"os"`
-	Arch string `json:"arch"`
+	OS   string   `json:"os"`
+	Arch string   `json:"arch"`
+	Env  []string `json:"env"`
+}
+
+func (t BuildTarget) GetEnvVariables() []string {
```

Comment:
```suggestion
func (t BuildTarget) EnvVariables() []string {
```
Removing `Get` would be more Go-idiomatic

Reply:
a6d6867d8e184927b2aa49feb335e2f4db29657e